### DISCUSSION TITLE
csv export

### DISCUSSION THREAD

**Discussion by JLT032** [yamafaktory/jql#90] [Ideas]
hey,

Thanks for what seems to be one excellent tool. 

What i'm looking for is a tool which can to this and also export to CSV (or SQL)

This because the .json i'm working on is so very json it's very hard to convert but your page actually documents the case using objects and arrays with cross references etc.

**Comment by yamafaktory** [comment]
Hi @commandline-be and thanks for the feedback.

`jql` aimed at only working on JSON (JSON -> JSON). I'd like to keep it simple with a clear scope.

However, I'm always opened to feedback / suggestions. `jq` does it for example, but in very limited way - see https://stedolan.github.io/jq/manual/:
> The input must be an array, and it is rendered as CSV with double quotes for strings, and quotes escaped by repetition.
